test2code/src/main/kotlin/com/epam/drill/test2code/coverage/CoverageTransport.kt

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2020 - 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.drill.test2code.coverage
+
+import com.epam.drill.common.agent.*
+import java.util.concurrent.atomic.*
+
+interface CoverageTransport {
+    fun send(message: String)
+
+    fun isAvailable(): AtomicBoolean
+}
+
+open class CoverageTransportImpl(
```

Comment:
I suggest renaming to `WebsocketCoverageTransport` since this is what we are going for
Although yes, this implementation is very much a mock now

Reply:
renamed.